src/pages/signin/SignInPageLayout/signInPageStyles.js/index.ios.js

```diff
@@ -0,0 +1,5 @@
+import styles from '../../../../styles/styles';
+
+const scrollViewContentContainerStyles = styles.flex1;
```

Comment:
@parasharrajat I'm not able to understand why flex1 was used for iOS.

Reply:
Not remember but I'm sure there was some UI thing. I will get back to you on this.

Reply:
Cool, take your time. Thanks!

Reply:
@rushatgabhane I looked into this one. I added this as minHeight: 100% makes the Login Page width become more the screen size and thus page will become scrollable even if we have enough space to fit the whole content.

Reply:
welp, we made iOS use minHeight 100% in this PR #7205 
This is the behaviour that we get after making the changes. It's same as using flex1 at least on the sim 😅

https://user-images.githubusercontent.com/29673073/149640142-7dcfdd8f-87a8-46fc-9ea9-17d02fb70a0f.mov

Reply:
Clear the cache and then run it. I don't remember but it i visible sometimes.

Reply:
Thanks for your help. Yep, it's a problem only sometimes so missed it during review.

It's a regression. I'll comment in original issue.